packages/shared/src/graphql/notifications.ts

```diff
@@ -41,6 +47,33 @@ export interface NotificationsData {
   notifications: Connection<Notification>;
 }
 
+export enum NotificationIcon {
+  DailyDev = 'DailyDev',
+  CommunityPicks = 'CommunityPicks',
+  Comment = 'Comment',
+  Upvote = 'Upvote',
+  Bell = 'Bell',
+}
+
+export const notificationIcon: Record<
+  NotificationIcon,
+  ComponentType<IconProps>
+> = {
+  [NotificationIcon.DailyDev]: DailyIcon,
+  [NotificationIcon.CommunityPicks]: DailyIcon,
+  [NotificationIcon.Comment]: DiscussIcon,
+  [NotificationIcon.Upvote]: UpvoteIcon,
+  [NotificationIcon.Bell]: BellIcon,
+};
+
+export const notificationDefaultTheme: Record<NotificationIcon, string> = {
```

Comment:
Lets move these to a utilities directory

Reply:
I agree, this is no way related to the graphql query. Let's move this.